File: stepik-union/src/main/java/org/stepik/plugin/projectWizard/idea/StepModuleBuilder.java
package org.stepik.plugin.projectWizard.idea;

import com.intellij.ide.highlighter.ModuleFileType;
import com.intellij.openapi.util.io.FileUtil;
import org.stepik.core.courseFormat.StepNode;
import org.jetbrains.annotations.NotNull;

public class StepModuleBuilder extends ModuleBuilderWithSrc {
    public StepModuleBuilder(String moduleDir, @NotNull StepNode stepNode) {
        String stepName = stepNode.getDirectory();
        setName(stepName);
        setModuleFilePath(FileUtil.join(moduleDir, stepName,
                stepName + ModuleFileType.DOT_DEFAULT_EXTENSION));
    }
}

File: hyperskill/src/main/kotlin/org/hyperskill/projectWizard/idea/StepModuleBuilder.kt
package org.hyperskill.projectWizard.idea

import com.intellij.ide.highlighter.ModuleFileType
import com.intellij.openapi.util.io.FileUtil
import org.stepik.core.courseFormat.StepNode

class StepModuleBuilder(moduleDir: String, stepNode: StepNode) : ModuleBuilderWithSrc() {
    init {
        val stepName = stepNode.directory
        name = stepName
        moduleFilePath = FileUtil.join(moduleDir, stepName,
                stepName + ModuleFileType.DOT_DEFAULT_EXTENSION)
    }
}
